modules/server/src/topics/errorlogtopic.cpp

```diff
@@ -112,6 +73,43 @@ void ErrorLogTopic::handleJson(const nlohmann::json& json) {
         ghoul::logging::LogManager::ref().removeLog(_log);
         _log = nullptr;
     }
+
+    if (event == UpdateLogLevel) {
+        ghoul::logging::LogManager::ref().removeLog(_log);
+        _log = nullptr;
+
+
+        std::string level = json.at(LogLevelKey).get<std::string>();
+        auto logLevel = ghoul::from_string<ghoul::logging::LogLevel>(level);
+        createLog(logLevel);
+    }
+}
+
+void ErrorLogTopic::createLog(ghoul::logging::LogLevel logLevel) {
+    if (_log) {
+        return;
+    }
```

Comment:
Is this check really needed? Seems like we currently do not risk calling the createLog function with the log already set. And if someone does, it might be intentional?